.dotnet/Azure.AI.OpenAI.Staging/Custom/Audio/AzureAudioClient.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using OpenAI.Audio;
+using System.ClientModel.Primitives;
+
+namespace Azure.AI.OpenAI.Staging.Audio;
+
+/// <summary>
+/// The scenario client used for audio operations with the Azure OpenAI service.
+/// </summary>
+/// <remarks>
+/// To retrieve an instance of this type, use the matching method on <see cref="AzureOpenAIClient"/>.
+/// </remarks>
+public partial class AzureAudioClient : AudioClient
+{
+    private readonly string _deploymentName;
+    private readonly Uri _endpoint;
+    private readonly string _apiVersion;
+
+    protected internal AzureAudioClient(
```

Comment:
Also just curious: what motivates making these protected?  That would enable further client inheritance, but do we have a scenario that would make us want to do that?  I ask just because of the old API design adage -- "you can add it, but you can never take it away again."

Reply:
Very fair point. I don't have any good reason right now; I'll make these plain internal.